--- solfege-2.1.3/po/Makefile.orig	2004-06-28 17:18:21.000000000 +0200
+++ solfege-2.1.3/po/Makefile	2004-07-06 19:03:41.072314512 +0200
@@ -47,8 +47,8 @@
 install-po:
 	echo $(POLANGUAGES)
 	for lang in $(POLANGUAGES); do \
-	  mkdir -p $(datadir)/locale/$$lang/LC_MESSAGES; \
-	  $(INSTALL_DATA) po/$$lang.mo $(datadir)/locale/$$lang/LC_MESSAGES/$(PACKAGE).mo; \
+	  mkdir -p $(DESTDIR)$(datadir)/locale/$$lang/LC_MESSAGES; \
+	  $(INSTALL_DATA) po/$$lang.mo $(DESTDIR)$(datadir)/locale/$$lang/LC_MESSAGES/$(PACKAGE).mo; \
 	done
 
 uninstall-po:
--- solfege-2.1.3/soundcard/Makefile.orig	2004-06-28 17:18:22.000000000 +0200
+++ solfege-2.1.3/soundcard/Makefile	2004-07-06 19:04:24.594698104 +0200
@@ -87,11 +87,11 @@
 
 install-soundcard:
 ifeq ($(ENABLE_OSS_SOUND),yes)
-	mkdir -p $(libdir)/$(PACKAGE)/$(VERSION)
+	mkdir -p $(DESTDIR)$(libdir)/$(PACKAGE)/$(VERSION)
 ifeq ($(SWIG_VERSION),1.1)
-	$(INSTALL_DATA) soundcard/solfege_c_midimodule.so $(libdir)/$(PACKAGE)/$(VERSION)/
+	$(INSTALL_DATA) soundcard/solfege_c_midimodule.so $(DESTDIR)$(libdir)/$(PACKAGE)/$(VERSION)/
 else
-	$(INSTALL_DATA) soundcard/_solfege_c_midi.so $(libdir)/$(PACKAGE)/$(VERSION)/
+	$(INSTALL_DATA) soundcard/_solfege_c_midi.so $(DESTDIR)$(libdir)/$(PACKAGE)/$(VERSION)/
 endif
 endif
 
--- solfege-2.1.3/graphics/Makefile.orig	2004-06-28 17:18:22.000000000 +0200
+++ solfege-2.1.3/graphics/Makefile	2004-07-06 19:04:45.132575872 +0200
@@ -22,8 +22,8 @@
 
 install-graphics:
 ifeq ($(WITH_GNOME),yes)
-	mkdir -p $(datadir)/pixmaps
-	$(INSTALL_DATA) graphics/solfege.png $(datadir)/pixmaps/
+	mkdir -p $(DESTDIR)$(datadir)/pixmaps
+	$(INSTALL_DATA) graphics/solfege.png $(DESTDIR)$(datadir)/pixmaps/
 endif
 
 uninstall-graphics:
--- solfege-2.1.3/Makefile.in.orig	2004-06-28 17:18:21.000000000 +0200
+++ solfege-2.1.3/Makefile.in	2004-07-06 19:07:14.873811744 +0200
@@ -124,17 +124,17 @@
 	rm -f $(clean-files)
 
 install: all mkinstalldirs install-po install-soundcard install-graphics
-	$(INSTALL_PROGRAM) run-solfege.py $(bindir)/solfege$(VERSION)
-	$(INSTALL_PROGRAM) run-solfege.py $(bindir)/solfege
-	$(INSTALL_DATA) solfege.gtkrc $(datadir)/$(PACKAGE)/$(VERSION)
-	$(INSTALL_DATA) default.config $(datadir)/$(PACKAGE)/$(VERSION)
-	-$(INSTALL_DATA) solfege.1 $(mandir)/man1
-	$(INSTALL_DATA) system.rc $(sysconfdir)/$(PACKAGE)$(MAJOR_VERSION).$(MINOR_VERSION)
+	$(INSTALL_PROGRAM) run-solfege.py $(DESTDIR)$(bindir)/solfege$(VERSION)
+	$(INSTALL_PROGRAM) run-solfege.py $(DESTDIR)$(bindir)/solfege
+	$(INSTALL_DATA) solfege.gtkrc $(DESTDIR)$(datadir)/$(PACKAGE)/$(VERSION)
+	$(INSTALL_DATA) default.config $(DESTDIR)$(datadir)/$(PACKAGE)/$(VERSION)
+	-$(INSTALL_DATA) solfege.1 $(DESTDIR)$(mandir)/man1
+	$(INSTALL_DATA) system.rc $(DESTDIR)$(sysconfdir)/$(PACKAGE)$(MAJOR_VERSION).$(MINOR_VERSION)
 ifeq ($(WITH_GNOME),yes)
-	mkdir -p $(datadir)/gnome/apps/Applications/
-	$(INSTALL_DATA) solfege.desktop $(datadir)/gnome/apps/Applications/
+	mkdir -p $(DESTDIR)$(applnkdir)
+	$(INSTALL_DATA) solfege.desktop $(DESTDIR)$(applnkdir)
 endif
-	$(PYTHON) tools/pcopy.py $(installfiles) $(datadir)/$(PACKAGE)/$(VERSION)/
+	$(PYTHON) tools/pcopy.py $(installfiles) $(DESTDIR)$(datadir)/$(PACKAGE)/$(VERSION)
 
 # FIXME add to install target when we know how to get the path to compileall.py
 compileall:
@@ -145,10 +145,10 @@
 endif
 
 mkinstalldirs:
-	mkdir -p $(mandir)/man1
-	mkdir -p $(bindir)
-	mkdir -p $(datadir)/$(PACKAGE)/$(VERSION)
-	mkdir -p $(sysconfdir)
+	mkdir -p $(DESTDIR)$(mandir)/man1
+	mkdir -p $(DESTDIR)$(bindir)
+	mkdir -p $(DESTDIR)$(datadir)/$(PACKAGE)/$(VERSION)
+	mkdir -p $(DESTDIR)$(sysconfdir)
 
 new-uninstall: uninstall-po uninstall-soundcard uninstall-graphics
 	rm $(addprefix $(datadir)/$(PACKAGE)/$(VERSION)/,$(install-files))
