--- solfege-3.6.4/configure.ac.orig	2006-12-08 11:16:31.000000000 +0100
+++ solfege-3.6.4/configure.ac	2006-12-10 21:33:51.459779366 +0100
@@ -126,9 +126,9 @@
 
 dnl Checks for libraries.
 if test "x$ENABLE_TUNER" = xyes; then
-    AC_DEFINE(ENABLE_TUNER)
+    AC_DEFINE(ENABLE_TUNER, , [the microphone experiment])
     AC_CHECK_LIB(rfftw, rfftw, [
-        AC_DEFINE(HAVE_FFTW)
+        AC_DEFINE(HAVE_FFTW, , [have rfftw library])
         LIBS="$LIBS -lfftw -lrfftw"], ,-lfftw)
 fi
 
--- solfege-3.6.4/soundcard/Makefile.orig	2006-12-08 11:16:31.000000000 +0100
+++ solfege-3.6.4/soundcard/Makefile	2006-12-10 22:01:39.994863763 +0100
@@ -25,7 +25,7 @@
 
 easy-dist-files += soundcard/solfege_c_midi_wrap.c
 
-CFLAGS:=$(CFLAGS) -I/usr/src/linux/include -Wall -pedantic -fPIC
+CFLAGS:=$(CFLAGS) -Wall -pedantic -fPIC
 CC:=$(CC) $(CFLAGS) -DHAVE_CONFIG_H 
 
 ifeq ($(ENABLE_OSS_SOUND),yes)
