--- solfege-3.12.0/solfege.desktop.orig	2008-11-08 11:35:31.000000000 +0100
+++ solfege-3.12.0/solfege.desktop	2008-11-16 10:16:59.984022119 +0100
@@ -1,7 +1,9 @@
 [Desktop Entry]
 Name=GNU Solfege
+Comment=Solfege eartraining program
+Comment[pl]=Program do ćwiczenia słuchu Solfege
 Exec=solfege
 Terminal=false
 Type=Application
 Icon=solfege.png
-Categories=GNOME;Application;AudioVideo;Audio;Education;
+Categories=GTK;GNOME;AudioVideo;Audio;Education;
