--- solfege-3.6.4/solfege.desktop.orig	2006-12-08 11:16:31.000000000 +0100
+++ solfege-3.6.4/solfege.desktop	2006-12-10 21:36:10.363695050 +0100
@@ -1,8 +1,10 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=GNU Solfege
+Comment=Solfege eartraining program
+Comment[pl]=Program do ćwiczenia słuchu Solfege
 Exec=solfege
 Terminal=false
 Type=Application
 Icon=solfege.png
-Categories=GNOME;Application;AudioVideo;Audio;Education;
+Categories=GTK;GNOME;AudioVideo;Audio;Education;
