diff -dur solfege-3.22.2.orig/solfege.desktop solfege-3.22.2/solfege.desktop
--- solfege-3.22.2.orig/solfege.desktop	2013-10-05 22:57:48.000000000 +0200
+++ solfege-3.22.2/solfege.desktop	2016-12-13 15:07:01.000000000 +0100
@@ -2,6 +2,8 @@
 Name=GNU Solfege
 GenericName=Ear training
 GenericName[de]=Gehörtraining
+Comment=Solfege eartraining program
+Comment[pl]=Program do ćwiczenia słuchu Solfege
 Exec=solfege
 Terminal=false
 Type=Application
