--- solfege-1.9.99/acinclude.m4.orig	2003-07-31 23:56:09.000000000 +0200
+++ solfege-1.9.99/acinclude.m4	2003-08-24 15:24:20.626449328 +0200
@@ -253,7 +253,7 @@
 
 AC_DEFUN([MY_SWIG_TEST],
   [
-  $SWIG -version > tmpswig 2&>tmpswig
+  $SWIG -version > tmpswig 2>tmpswig
   if grep "SWIG Version 1.3" tmpswig
   then
     [$1="1.3"]
--- solfege-1.9.99/configure.ac.orig	2003-07-31 23:56:09.000000000 +0200
+++ solfege-1.9.99/configure.ac	2003-08-24 15:30:01.854574768 +0200
@@ -124,9 +124,9 @@
 
 dnl Checks for libraries.
 if test x"$ENABLE_TUNER" = x"yes"; then
-    AC_DEFINE(ENABLE_TUNER)
+    AC_DEFINE(ENABLE_TUNER, , [the microphone experiment])
     AC_CHECK_LIB(rfftw, rfftw, [
-        AC_DEFINE(HAVE_FFTW)
+        AC_DEFINE(HAVE_FFTW, , [have rfftw library])
         LIBS="$LIBS -lfftw -lrfftw"], ,-lfftw)
 fi
 
