diff -dur -x '*.orig' solfege-3.22.2.orig/configure.ac solfege-3.22.2/configure.ac
--- solfege-3.22.2.orig/configure.ac	2013-10-05 22:57:48.000000000 +0200
+++ solfege-3.22.2/configure.ac	2016-12-13 15:08:20.000000000 +0100
@@ -144,9 +144,9 @@
 
 dnl Checks for libraries.
 if test "x$ENABLE_TUNER" = xyes; then
-    AC_DEFINE(ENABLE_TUNER)
+    AC_DEFINE(ENABLE_TUNER, , [the microphone experiment])
     AC_CHECK_LIB(rfftw, rfftw, [
-        AC_DEFINE(HAVE_FFTW)
+        AC_DEFINE(HAVE_FFTW, , [have rfftw library])
         LIBS="$LIBS -lfftw -lrfftw"], ,-lfftw)
 fi
 
diff -dur solfege-3.23.4.orig/solfege/soundcard/Makefile solfege-3.23.4/solfege/soundcard/Makefile
--- solfege-3.23.4.orig/solfege/soundcard/Makefile	2016-06-24 22:18:29.000000000 +0200
+++ solfege-3.23.4/solfege/soundcard/Makefile	2016-12-13 15:30:59.000000000 +0100
@@ -23,7 +23,7 @@
 	solfege/soundcard/winmidi.dsp solfege/soundcard/winmidi.dsw solfege/soundcard/winmidi.def \
 	$(wildcard solfege/soundcard/tests/*.py)
 
-CFLAGS:=$(CFLAGS) -I/usr/src/linux/include -Wall -pedantic -fPIC -I.
+CFLAGS:=$(CFLAGS) -Wall -pedantic -fPIC -I.
 CC:=$(CC) $(CFLAGS) -DHAVE_CONFIG_H 
 
 ifeq ($(ENABLE_OSS_SOUND),yes)
